### PR TITLE
add table to show supported frameworks in Sauce Cloud

### PR DESCRIPTION
**Pull request by dpgraham**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by sweeneyskirt-sl — COMMENTED** [review]
Just one note. :)

File: docs/testrunner-toolkit.md [hunk 1]
```diff
@@ -41,6 +41,50 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
 :::
 
+<!--CLOUD FRAMEWORKS GO HERE. This markdown is generated from the test-composer project. Do not edit manually -->
+<!--START_AUTO_GENERATED_TABLE-->
+
+### Supported frameworks in Sauce Cloud
+
+<Tabs 
+    defaultValue="cypress"
+    values={[{"label":"Cypress","value":"cypress"},{"label":"Playwright","value":"playwright"},{"label":"TestCafe","value":"testcafe"}]}><TabItem value="cypress">
```
Comment (spider-sauce) [inline docs/testrunner-toolkit.md:51]:
```suggestion
### Supported frameworks in Sauce Cloud
<Tabs
  defaultValue="cypress"
  values={[
    {label: 'Cypress', value: 'cypress'},
    {label: 'Playwright', value: 'playwright'},
    {label: 'TestCafe', value: 'testcafe'},
    {label: 'Puppeteer', value: 'puppeteer'},
  ]}>
  
```

File: docs/testrunner-toolkit.md [hunk 2]
```diff
@@ -41,6 +41,50 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
+
```
Comment (spider-sauce) [inline docs/testrunner-toolkit.md:52]:
```suggestion
<TabItem value="cypress">

```

**Review by spider-sauce — COMMENTED** [review]
@dpgraham I added some fixes for the  `Tab` feature. Do we want to add the `puppeteer`  tab for the `docker` mode as well?

**Review by sweeneyskirt-sl — APPROVED** [review]

File: docs/testrunner-toolkit.md [hunk 3]
```diff
@@ -41,6 +41,50 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
+<TabItem value="playwright">
+
+|Playwright Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|1.7.2|Windows 10|Playwright Chromium, Playwright Firefox, Playwright Webkit|
+
+</TabItem>
+<TabItem value="puppeteer">
+
+|Puppeteer Version|Supported Platforms|Supported Browsers|
```
[the remark this answers is not in the document]
Comment (dpgraham) [reply docs/testrunner-toolkit.md:74]: @sweeneyskirt-sl yeah it's a placeholder until we support Puppeteer in our cloud. Do you think I should keep it there or just take it out?

File: docs/testrunner-toolkit.md [hunk 4]
```diff
@@ -41,6 +41,56 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
+|-----|-----|-----|
+|5.7.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
+|0.2.3|Windows 10|Chrome, Firefox, MicrosoftEdge|
+
+</TabItem>
+<TabItem value="playwright">
+
+|Playwright Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|1.7.2|Windows 10|Playwright Chromium, Playwright Firefox, Playwright Webkit|
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:69]: Don't think we need to enumerate the browsers here with the `playwright` prefix (despite it being the embedded browsers that come with playwright).


Also, that's the runner version listed there, not the playwright version 🤔

File: docs/testrunner-toolkit.md [hunk 5]
```diff
@@ -41,6 +41,56 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
+    {label: 'Playwright', value: 'playwright'},
+    {label: 'TestCafe', value: 'testcafe'},
+    {label: 'Puppeteer', value: 'puppeteer'},
+  ]}>
+  
+<TabItem value="cypress">
+
+|Cypress Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|5.7.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:61]: That's the runner version though, not the cypress version 🤔

File: docs/testrunner-toolkit.md [hunk 6]
```diff
@@ -41,6 +41,56 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
...
+|Puppeteer Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+
+
+</TabItem>
+<TabItem value="testcafe">
+
+|TestCafe Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|0.4.0|macOS 11.00|Chrome, Firefox, MicrosoftEdge, Safari|
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:83]: Same here, it's the runner versions listed, not the testcafe versions 🤔

File: docs/testrunner-toolkit.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sweeneyskirt-sl) [reply docs/testrunner-toolkit.md:74]: Hey @dpgraham -- I think @spider-sauce actually added Puppeteer to the list of tabs above. If you don't want it to show up just yet, I would got ahead and comment it out for now. Just my 2¢.

File: docs/testrunner-toolkit.md [hunk 7]
```diff
@@ -45,48 +45,52 @@ Each docker image tag is the 'latest' image that supports the specific framework
 <!--START_AUTO_GENERATED_TABLE-->
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit.md:45]: Hey @dpgraham -- Just curious about this note? It does not appear that anything between the `start ---end` comments is auto-generated? Are these comments leftover from an earlier version or do they mean something else?

Comment (dpgraham) [reply docs/testrunner-toolkit.md:45]: @sweeneyskirt-sl this just means that the table is generated elsewhere and copy-pasted here. I probably could have worded it better.

**Pull request merged**